src/components/PageElements/PageElements.tsx

```diff
@@ -100,90 +109,125 @@ const PageElements: React.FC<PageElementProps> = ({
     return (
       <Form>
         {visibleElements.map((state) => {
-          const {
-            element,
-            isChanged,
-            isChangeRequest,
-            previousApplicationResponse,
-            latestApplicationResponse,
-          } = state
-
-          const props = {
-            canEdit: !!canEdit,
-            linkToPage: `/application/${serial}/${sectionCode}/Page${pageNumber}`,
-            summaryProps: getSummaryViewProps(element),
-            latestApplicationResponse: latestApplicationResponse,
-            previousApplicationResponse: previousApplicationResponse,
-          }
+          const { element, isChanged, isChangeRequest, latestApplicationResponse } = state
 
-          const changedQuestionResponse = !isChangeRequest && !isChanged
+          const isResponseUpdated = !!isChangeRequest || !!isChanged
+          const canApplicantEdit = isUpdating ? isResponseUpdated && canEdit : canEdit
```

Comment:
is updating is set to isChangeRequest and passed through via props twice, can we just use isChangeRequest ?

Reply:
Also this logic here seems easier as `canEdit && (!!isChangeRequest || !!isChanged)`

Reply:
Sorry, didn't understand the first comment. Also, when not a change (`!isUpdating`) we still want to check `canEdit`, which is the case of `canEdit=true`, `isChangeRequest=false` and `isChanged=false` would return false...

Reply:
>Also, when not a change (!isUpdating) we still want to check canEdit, which is the case of canEdit=true, isChangeRequest=false and isChanged=false would return false...

Hmm, sorry for confusion, I started this comment thinking `isUpdating` is not needed for this check, and forgot that we show edit icon in summary view differently based on draft review of application as change request differently to when we show the edit button on first time draft. your change looks good., but a comment would be helpful here

Reply:
Added a comment